package.json

```diff
@@ -35,5 +37,8 @@
   "lint-staged": {
     "src/**/*.js": "eslint --fix",
     "src/**/*": "prettier --write"
+  },
+  "dependencies": {
+    "node-fetch": "^2.6.1"
```

Comment:
Curious, is this required to be a dependency vs dev dependency?

Needs an exact version.

Reply:
I'm a confused on what's a dependency vs dev dependency. But, I included this in dependencies because we need it in production to make the API call.

Reply:
Yeah, it's a bit tricky. Usually a dependency is a is something that will be imported/required/etc during the website runtime. But because we build everything first, we only have dev dependencies. Those are available at `yarn build`.

Netlify Functions are a little different. Try putting it as a devDependency and seeing if your import of it still works. I'm not sure how Netlify handles this really.

Reply:
Will do